msu/ui/ui.nut

```diff
@@ -0,0 +1,31 @@
+::MSU.UI <- {
+	Connections = [],
+	Callbacks = [],
+	JSConnection = null,
+
+	function registerConnection( _connection )
+	{
+		this.Connections.push(_connection);
+	}
+
+	function connect()
+	{
+		foreach (connection in this.Connections)
+		{
+			connection.connect();
+		}
+		this.Connections.clear();
+		foreach (callback in this.Callbacks)
+		{
+			callback();
+		}
+		this.Callbacks.clear();
+	}
+
+	function addAfterConnectionCallback( _function )
```

Comment:
I don't understand what is AfterConnectionCallback. It seems that the callbacks are all called in connect() and then cleared.

Reply:
I see, then the name is rather misleading. It seems to suggest these callbacks are AFTER connect. Suggest renaming to `addOnConnectCallback`

Reply:
I mean, it is after connect() as per the function above, but sure